x11-misc/revelation/revelation-0.5.4.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_8 )
+
+inherit python-single-r1 meson xdg
+
+DESCRIPTION="A password manager for GNOME"
+HOMEPAGE="https://revelation.olasagasti.info/ https://github.com/mikelolasagasti/revelation"
+SRC_URI="https://github.com/mikelolasagasti/revelation/releases/download/${P}/${P}.tar.xz"
+
+LICENSE="GPL-2+"
```

Comment:
Github is really bad at showing license files, you want to check the source files and their headers that are a bit more precise:
https://github.com/mikelolasagasti/revelation/blob/2a0b42276377fa7c44ac2e510f86ac33d36aa6b6/src/lib/util.py